libraries/ms-helper/test/test_thermistor.c

```diff
@@ -0,0 +1,41 @@
+#include "log.h"
+#include "test_helpers.h"
+#include "thermistor.h"
+
+void setup_test(void) {}
+
+void teardown_test(void) {}
+
+void test_thermistor(void) {
+  GPIOAddress gpio_addr = {
+    .port = GPIO_PORT_A,
+    .pin = 4,
+  };
+
+  GPIOSettings gpio_settings = {
+    .direction = GPIO_DIR_IN,
+    .state = GPIO_STATE_LOW,
+    .resistor = GPIO_RES_NONE,
+    .alt_function = GPIO_ALTFN_ANALOG,
+  };
+
+  ThermistorSettings settings = {
+    .sibling_resistance = 10000000,
+    .adc_channel = ADC_CHANNEL_4,
+  };
+
+  ThermistorStorage storage;
+
+  // initialize gpio pin
+  gpio_init_pin(&gpio_addr, &gpio_settings);
+
+  // initialize the channel
+  adc_init(ADC_MODE_CONTINUOUS);
+
+  // for some reason this test is being included in x86 tests
```

Comment:
It fails because the ADC isn't implemented on x86. All unit tests are included in both platforms by default. You need to explicitly disable tests if you want to change that.

Reply:
How would I go about doing so?

Reply:
You need to use `$(T)_EXCLUDE_TESTS` in `rules.mk`. See `ms-common` or `plutus` for examples.